hydra-core/spec/test_app_templates/Gemfile.extra

```diff
@@ -1,2 +1,3 @@
 gem 'rspec-rails', '~> 3.1', group: :test
 gem 'rspec-its'
+gem 'active-fedora', :github => "projecthydra/active_fedora"
```

Comment:
How do you usually handle this? You tend to release hydra-head in accordance with releases of AF, right? Are hydra-head features that depend on AF always a version behind?